assignments/callbacks.js

```diff
@@ -39,26 +39,42 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 */
 
 
-function getLength(arr, cb) {
+function getLength(arr,cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
+getLength(items, howLong => console.log(howLong));
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length-1])
 }
 
+last(items, finalItem => console.log(finalItem));
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x+y);
 }
+// const x = 5;
+// const y = 2;
+sumNums(x, y, sum => console.log(sum));
```

Comment:
Instead of declaring constants and passing the variables into a function, you could also write the function like:
```
sumNums(5, 2, sum => console.log(sum));
```

You can use this same idea when you call `multiplyNums()` and `contains()` down below '' '